students/elaine_x/session06/mailroom4.py

```diff
@@ -0,0 +1,165 @@
+'''
+##########################
+#Python 210
+#Session 06 - Mailroom Part 4
+#Elaine Xu
+#Feb 20, 2019
+###########################
+'''
+import sys
+#import datetime
+#import operator
+
+"""modifying this function for unit test
```

Comment:
not sure why you are doing giant comment out blocks, just remove the unused code.